.travis.yml

```
@@ -16,8 +16,15 @@ script:
 - npm run travis:verify
 after_script:
 - "./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT"
+after_success:
+curl -sSL https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master/src/bootstrap.sh | bash -s
```

this should not be a separate line, and if it is, it can't be undented like this (invalid yaml)...might be better to do this

```suggestion
- curl -sSL https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master/src/bootstrap.sh | bash -s
```